scapy/layers/inet6.py

```diff
@@ -43,33 +43,33 @@
     # Workaround for https://bitbucket.org/secdev/scapy/issue/5119
     socket.IPPROTO_IPIP = 4
 
-from scapy.arch import get_if_hwaddr
-from scapy.as_resolvers import AS_resolver_riswhois
-from scapy.base_classes import Gen
-from scapy.compat import chb, orb, raw, plain_str
-from scapy.config import conf
-import scapy.consts
+from scapy.arch import get_if_hwaddr  # noqa: E402
+from scapy.as_resolvers import AS_resolver_riswhoi  # noqa: E402s
+from scapy.base_classes import Ge  # noqa: E402n
```

Comment:
Few glitches over here

Reply:
Thanks!
I managed to fix it before boarding =)